electron/app/copyCaptain/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as sqlite3 from 'sqlite3';
+import * as fs from 'fs';
+import copy from 'recursive-copy';
+import log from 'electron-log';
+import { Directories } from './definitions';
+
+// TODO: possibly rethink if this needs to be a class
+// Original idea is that this would take a db path as a instance variable
+// Would also take things like if { overwrite: true } should be used
+export class CopyCaptain {
+  constructor() {
```

Comment:
this should take input and output, and be used for both contacts and chat.dbs

Reply:
Scratch that, just make a file that is called "copy utils" and call them in order